perma_web/perma/settings/deployments/settings_common.py

```diff
@@ -137,6 +137,7 @@
     # built in apps
     'django.contrib.auth',
     'django.contrib.contenttypes',
+    'django.contrib.postgres',
```

Comment:
Is this an app in the Django sense?

Reply:
Weird, right? It's required, for a person to use `OpClass` https://docs.djangoproject.com/en/4.1/ref/contrib/postgres/indexes/#opclass-expressions

Reply:
TIL! 👍